drizzlepac/align.py

```diff
@@ -441,13 +440,16 @@ def perform_align(input_list, catalog_list, num_sources, archive=False, clobber=
                         return alignment_table
                 else:
                     log.info("{} Cross matching and fitting {}".format("-" * 20, "-" * 47))
-                    imglist = copy.deepcopy(orig_imglist)  # reset imglist to pristine state
-
                     log.info(
                         "{} Catalog {} matched using {} {}".format("-" * 18,
                                                                    catalog_name,
                                                                    algorithm_name, "-" * 18))
                     try:
+                        alignment_table.configure_fit()
+                        print("####\n# Running configure fit for AlignmentTable to use: \n")
+                        print([img.wcs for img in alignment_table.imglist])
+                        print("####\n")
+
```

Comment:
Please change these print statements to log statements of some flavor (info or debug).

Reply:
Fixed in e61b53a.